src/blog/announcing-tezos-truffle-integration-multi-blockchain-strategy.md

```diff
@@ -0,0 +1,22 @@
+![Truffle Teams Deployments Banner](/img/blog/unwrap-the-corda-flavored-ganache-beta/corda-ganache-beta-blog-banner.png)
+
+If you've been following Truffle's development, you'll notice that our tools already support a handful of exciting platorms: [Ethereum](#), [Quorum](#), [Axoni](#), Hyperledger [Fabric-EVM](#), Hyperledger [Sawtooth](#) & [Burrow](#), and [GoChain](#) to name a few. What's common amongst these chains is that they're all based on the Ethereum Virtual Machine. As you might expect, Ethereum and its virtual machine are very near and dear to our hearts -- it's where we got our start, and where we're the most competant. So today I'd like to announce something new, the first step on a path toward Truffle's longer term strategy: **Tezos**. 
+
+**I'm excited to announce we've added experimental Tezos integration into the Truffle command line tool.** This means that right now, at this very moment, you can [download a Tezos-specific version of Truffle](#) and start building smart contracts on Tezos networks using the time-tested tooling you already know and love. And if you're a Tezos native who got _your_ start building on top of the Tezos platform, then this means professional-grade tooling has finally arrived. 
+
+We chose Tezos as our first non-EVM chain because of its blossoming developer community. In our position as tools builders, we go where the market takes us: If enough people ask for Tezos support, we listen. This market demand was coupled with an active and interested group of core developers, and a blockchain architecture that's similar to Ethereum. Ultimately, Tezos is "EVM-like". This makes it a great candidate for integration, fitting into Truffle like a glove. 
```

Comment:
```suggestion
We chose Tezos as our first non-EVM chain because of its blossoming developer community. In our position as tool builders, we go where the market takes us: If enough people ask for Tezos support, we listen. This market demand was coupled with an active and interested group of core developers, and a blockchain architecture that's similar to Ethereum. Ultimately, Tezos is "EVM-like". This makes it a great candidate for integration, fitting into Truffle like a glove. 
```

?